lib/location.js

```diff
@@ -261,9 +275,16 @@ var LocationState = State.extend(function (self, name, opts) {
         return '\n' + number + '. ' + address.label;
     };
 
-    self.generate_footer = function() {
-        return '\n' + 'n. '+self.next_text + '\n' + 'p. '+self.previous_text;
-
+    self.generate_footer = function(content) {
+        var options = [];
+        if (self.metadata.current_page >= self.metadata.pages.length && self.retry_text){
```

Comment:
r. Retry should replace the n. Next on the last page only. The current_page > pages.length check is to see whether the last page is being displayed so that the retry option can replace the next option